pkgs/development/python-modules/bsddb3/default.nix

```diff
@@ -23,7 +23,15 @@ buildPythonPackage rec {
 
   buildInputs = [ pkgs.db ];
 
-  doCheck = pythonOlder "3.12"; # distutils usage
+  broken = pythonOlder "3.6";
```

Comment:
By my reading it's not actually broken until 3.10

Reply:
Maybe yeah, I checked again and it is recommended to replace the dependency if possible for Python 3.6+ : 
![image](https://github.com/NixOS/nixpkgs/assets/14914796/2a574c5d-708f-4c7a-b47e-70cffb4f2436)

In the case, where we want to maximize compatibility, it's most likely a good idea to keep it until Python 3.10.

![image](https://github.com/NixOS/nixpkgs/assets/14914796/ad9e470f-0751-4637-a7a1-a50506a29497)

So you're right, I will update this.